src/components/SubPage/convert-data.ts

```diff
@@ -127,7 +127,7 @@ export const unpackParentLineFromFrontMatter = (data: ParentalLineFrontmatter):
             data
                 .genetic_modifications
         );
-    const thumbnailImage = data.thumbnail_image;
+    const thumbnailImage = data.images_and_videos?.images?.[0]?.image?.childImageSharp?.gatsbyImageData || null;
```

Comment:
personally I don't like all of these `?` because it seems like we don't have a good handle on our own data. if the user doesn't upload any images, is there also no `images_and_videos` field? and if you request the data and it isn't there, does it have `images_and_videos: undefined`?